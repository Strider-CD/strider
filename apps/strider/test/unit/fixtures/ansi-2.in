[0m Git clone complete
[35mstrider $[0m [33mnpm install[0m
0
