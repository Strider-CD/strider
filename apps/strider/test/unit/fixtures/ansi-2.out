 Git clone complete
strider $ npm install
0
